tests/pkg_libcoap/main.c

```diff
@@ -18,6 +18,6 @@
 
 int main(void)
 {
-    puts("Libcoap compiled!");
+    puts("SUCCESS");
```

Comment:
I think its good to let the user know what was successful, hence `[SUCCESS] libcoap compiled!` might be more appropriate.